### ISSUE TITLE
Problem with running on M1 Mac

### ISSUE THREAD

**Issue by irinatu** [ResearchIT/IGV-ScanFold#41]
Hi,
I tried to run ScanFold on M1 Mac and have below information about the missing folder
<img width="1604" alt="Screenshot 2022-11-16 at 21 30 15" src="https://user-images.githubusercontent.com/12935885/202288101-5766fdb5-3d7d-4461-abf2-af4d39af957d.png">

**Comment by njbooher** [comment]
I don't have an M1 Mac available to test with, and I don't think Github Actions supports building packages for M1 yet.

**Comment by njbooher** [comment]
might just be a new macos issue. will investigate more

```
['/private/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/AppTranslocation/E53E2B1F-DC40-4B8E-8914-7AFA6D9E3746/d/IGV-ScanFold.app/Contents/scanfold/ScanFold-Scan_IGV.dist/ScanFold-Scan_IGV', '-i', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/scanfoldinput2382923032006302084.fa', '-r', '30', '-s', '10', '-t', '37', '-w', '120', '-d', 'forward', '--start', '0', '--name', 'test', '-type', 'mono', '--scan_out_path', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/outputaLoq4m.scan-out.tsv', '--fasta_file_path', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/inputoE1qCf.fasta', '--fasta_index', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/inputyQ2Gxb.fasta.fai', '--algo', 'rnafold']
Done['/private/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/AppTranslocation/E53E2B1F-DC40-4B8E-8914-7AFA6D9E3746/d/IGV-ScanFold.app/Contents/scanfold/ScanFold-Scan_IGV.dist/ScanFold-Scan_IGV', '-i', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/scanfoldinput16667978366512313586.fa', '-r', '30', '-s', '10', '-t', '37', '-w', '120', '-d', 'forward', '--start', '0', '--name', 'test', '-type', 'mono', '--scan_out_path', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/outputDdV1KD.scan-out.tsv', '--fasta_file_path', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/input9A6uy4.fasta', '--fasta_index', '/var/folders/r9/qxt2c9js3w7fwns2mq1hvf6x16zq89/T/scanfold-results8307767269669066262/inputEASxLI.fasta.fai', '--algo', 'rnastructure']
Done
```

**Comment by njbooher** [comment]
Please try opening the Terminal app and using the following to Download and run the software:

```
cd ~/Downloads/
curl -L -O https://github.com/ResearchIT/IGV-ScanFold/releases/download/v0.2.3/IGV-ScanFold-macos-v0.2.3.zip
unzip IGV-ScanFold-macos-v0.2.3.zip
open -a IGV-ScanFold.app
```

Please let me know if this resolves this issue.

**Comment by rjandr** [comment]
That fix works only in a Rosetta conda environment - replicated the error and confirmed the fix on an M1 MacOS version 13.2.1 
1. Download and install miniconda (https://docs.conda.io/projects/continuumio-conda/en/latest/user-guide/install/macos.html)
2. Create a Rosetta conda environment (https://github.com/conda-forge/miniforge/issues/165 ). 
3. Activate Rosetta environment
4. Open IGV-ScanFold
```
CONDA_SUBDIR=osx-64 conda create -n rosetta python   # create a new environment called rosetta with intel packages.
conda activate rosetta
python -c "import platform;print(platform.machine())"
conda config --env --set subdir osx-64  # make sure that conda commands in this environment use intel packages
cd ~/Downloads/
curl -L -O https://github.com/ResearchIT/IGV-ScanFold/releases/download/v0.2.3/IGV-ScanFold-macos-v0.2.3.zip
unzip IGV-ScanFold-macos-v0.2.3.zip
open -a IGV-ScanFold.app
```
